glass/src/lib/native/cpp/support/IniSaverInfo.cpp

```diff
@@ -140,12 +138,12 @@ bool NameInfo::InputTextName(const char* label_id, ImGuiInputTextFlags flags) {
   return ImGui::InputText(label_id, m_name, sizeof(m_name), flags);
 }
 
-bool OpenInfo::ReadIni(wpi::StringRef name, wpi::StringRef value) {
+bool OpenInfo::ReadIni(std::string_view name, std::string_view value) {
   if (name != "open") {
     return false;
   }
   int num;
-  if (value.getAsInteger(10, num)) {
+  if (wpi::StringRef{value}.getAsInteger(10, num)) {
```

Comment:
We could make a std::string_view equivalent of std::stoi() or use https://en.cppreference.com/w/cpp/utility/from_chars.

Reply:
Doing pretty much option 2 in #3394.  from_chars is not supported on the Rio GCC version.